src/asynqp/__init__.py

```diff
@@ -19,8 +19,12 @@
 def connect(host='localhost',
             port=5672,
             username='guest', password='guest',
-            virtual_host='/', *,
-            loop=None, sock=None, **kwargs):
+            virtual_host='/',
+            *,
+            loop=None,
+            sock=None,
+            nodelay=True,
```

Comment:
I'd prefer a more general API for setting options on the socket. Perhaps the client could pass in a list of tuples: `connect(..., sockopt=[(socket.IPPROTO_TCP, socket.TCP_NODELAY, 1)]`. Then we iterate through the list and set each option on the socket.

We also need to make sure there aren't any unpleasant surprised when you combine this parameter with the `sock` parameter.